style.css

```diff
@@ -1,3 +1,58 @@
-body {
+/* body {
   background-color: white;
+} */
+
+#bookss {
+  /* border: 2px red solid; */
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.